Makefile.am

```diff
@@ -33,7 +33,7 @@ endif
 
 libreadstat_la_CFLAGS = -Wall
 libreadstat_la_LIBADD = -llzma -lz @EXTRA_LIBS@
-libreadstat_la_LDFLAGS =
+libreadstat_la_LDFLAGS = -no-undefined
```

Comment:
I'm not sure what that really does and whether this also makes sense on non-Windows platforms. Without it nothing compiled...